source/ns_cmdline.c

```diff
@@ -1401,20 +1401,22 @@ static void replace_variable(char *str, cmd_variable_t* variable_ptr)
     const char* name = variable_ptr->name_ptr;
     int name_len = strlen(variable_ptr->name_ptr);
     char* value;
+    char valueLocal[11];
     if (variable_ptr->type == VALUE_TYPE_STR) {
       value = variable_ptr->value.ptr;
     } else {
-      value = MEM_ALLOC(6);
-      int written = snprintf(value, 6, "%d", variable_ptr->value.i);
+      value = valueLocal;
+      int written = snprintf(value, 11, "%d", variable_ptr->value.i);
```

Comment:
This has invalid two char indentation and compiler warning for unused ```written``` variable

Reply:
good catch. We have to improve CI to catch these more carefully..

Reply:
fixed.